BUFR_TableD_en_40.csv

```diff
@@ -793,3 +793,169 @@ Category,CategoryOfSequences_en,FXY1,Title_en,SubTitle_en,FXY2,ElementName_en,El
+40,Additional satellite report sequences,340021,(Altimeter model values),,010089,Total geocentric ocean tide height (solution 2),,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,010090,Long period tide height,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,010092,Solid earth tide height,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,010093,Geocentric pole tide height,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,010098,Loading tide height geocentric ocean tide solution 1,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,010099,Loading tide height geocentric ocean tide solution 2,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,010100,Non-equilibrium long period tide height,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,025127,Inverted barometer correction,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,040014,High-frequency fluctuations of the sea-surface topography correc,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,001030,Numerical model identifier,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,010085,Mean sea-surface height,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,001030,Numerical model identifier,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,010085,Mean sea-surface height,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,010086,Geoid's height,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,010096,Mean dynamic topography,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,010103,Mean dynamic topography accuracy,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,021169,Ice presence indicator,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,013055,Intensity of precipitation,,,Operational
+40,Additional satellite report sequences,340021,(Altimeter model values),,025165,Ionospheric correction from model on specific band,,,Operational
+40,Additional satellite report sequences,340022,(Altimeter main values),,025095,Altimeter state flag,,,Operational
+40,Additional satellite report sequences,340022,(Altimeter main values),,040023,Auxiliary altimeter state flags,,,Operational
+40,Additional satellite report sequences,340022,(Altimeter main values),,008074,Altimeter echo type,,,Operational
+40,Additional satellite report sequences,340022,(Altimeter main values),,340023,Yyy,,,Operational
```

Comment:
From here to line 873 please replace descriptor titles with those from proposal:

Table references | Element name
-- | --
3-40-019 | Satellite general values
3-40-020 | General radiometer values
3-40-021 | Altimeter model values
3-40-022 | Altimeter main values